### DISCUSSION TITLE
今天摇骰子怎么一直失败

### DISCUSSION THREAD

**Discussion by ry424515050** [BurnHair/docs#128] [分享讨论]
![Uploading 277a15fda1dd9a246113d28dd1b3c015.png…]()

**Comment by zhangdeshuai409930360** [comment]
me too~

**Comment by BennyThink** [comment]
是一个意料之中的事情～详情请见更新之后的服务条款。

**Reply by GinWU05** [reply]
去网站的[设置](https://burn.hair/setting)，绑定tg账号。 使用这个tg bot的 `/dice`摇骰子。
[https://t.me/burn_hair_bot](https://t.me/burn_hair_bot)
